aws/resource_aws_cognito_user_pool.go

```diff
@@ -131,6 +131,7 @@ func resourceAwsCognitoUserPool() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				MaxItems: 1,
+				Computed: true,
```

Comment:
Setting `Computed` for an attribute means that Terraform can't detect drift for unconfigured attributes. In this case the prefer schema change is to add a `DiffSuppressFunc` to ignore the case where Terraform thinks our schema default was a change when it's not, given the default API response.
```suggestion
				DiffSuppressFunc: suppressMissingOptionalConfigurationBlock,
```